### ISSUE TITLE
Feature Request: WAFv2 IP Set Resource

### ISSUE THREAD

**Issue by bflad** [terraform-providers/terraform-provider-aws#11173]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

In November 2019, AWS released a new version of the WAF API, [WAFv2](https://docs.aws.amazon.com/waf/latest/APIReference/Welcome.html), which offers improved functionality over the previous WAF API ("WAF Classic") such as [Managed Rules](https://github.com/terraform-providers/terraform-provider-aws/issues/11046) and WAF Capacity Units. This new API requires separate Terraform resource implementations from the previous resource implementations.

### New or Affected Resource(s)

* aws_wafv2_ip_set

### Potential Terraform Configuration

```hcl
resource "aws_wafv2_ip_set" "example" {
  addresses          = [""]
  description        = ""
  ip_address_version = ""
  name               = ""
  scope              = ""
  tags               = {}
}
```

### References

- Requires: https://github.com/terraform-providers/terraform-provider-aws/pull/11172
- API Reference: https://docs.aws.amazon.com/waf/latest/APIReference/API_Operations_AWS_WAFV2.html
- SDK Reference: https://docs.aws.amazon.com/sdk-for-go/api/service/wafv2/

**Comment by tsabbay** [comment]
This entire wafv2 is not baked!

**Comment by anGie44** [comment]
The fix for this has been merged and will release with version 2.63.0 of the Terraform AWS Provider, expected in this week's release.
